src/_zkapauthorizer/_plugin.py

```diff
@@ -256,7 +271,13 @@ async def setup_replication():
             await setup_tahoe_lafs_replication(tahoe)
             # And then turn replication on for the database connection already
             # in use.
-            self._add_replication_service(store)
+            client = get_tahoe_client(self.reactor, node_config)
+            mutable = get_replica_rwcap(node_config)
+            uploader = get_tahoe_lafs_direntry_uploader(client, mutable)
+            private_conn = _open_database(
+                partial(_connect, node_config.get_private_path(CONFIG_DB_NAME))
+            )
+            self._add_replication_service(store, private_conn, uploader)
```

Comment:
Maybe the duplication between `_get_store` and this could be factored out (esp. if we're going to change how `_get_store` works).

Reply:
e6f0cea